common/djangoapps/student/tests/test_reset_password.py

```diff
@@ -287,6 +288,35 @@ def test_password_reset_fail(self):
         self.assertEqual(resp.status_code, 200)
         self.assertFalse(User.objects.get(pk=self.user.pk).is_active)
 
+    @override_settings(PASSWORD_MIN_LENGTH=2)
+    @override_settings(PASSWORD_MAX_LENGTH=10)
+    @ddt.data(
+        {
+            'password': '1',
+            'error_message': 'Password: Invalid Length (must be 2 characters or more)',
+        },
+        {
+            'password': '01234567891',
+            'error_message': 'Password: Invalid Length (must be 10 characters or fewer)'
```

Comment:
@schenedx I can work on regex. But I believe this way its more readable, with regex simple code will become complex and if we change any of message we need to change regex too.